tests/integration/conftest.py

```diff
@@ -69,40 +162,25 @@ def user_config():
     return config
 
 
-# Raw sample datasets
-@pytest.fixture(scope="module")
-def sample_data_decennial_census():
-    return _load_sample_data("decennial_census")
-
-
-@pytest.fixture(scope="module")
-def sample_data_american_community_survey():
-    return _load_sample_data("american_community_survey")
-
-
-@pytest.fixture(scope="module")
-def sample_data_current_population_survey():
-    return _load_sample_data("current_population_survey")
-
-
-@pytest.fixture(scope="module")
-def sample_data_women_infants_and_children():
-    return _load_sample_data("women_infants_and_children")
+# Un-noised 1040
+@pytest.fixture(scope="session")
+def formatted_1040_sample_data():
+    formatted_1040 = generate_taxes_1040(
+        seed=SEED, year=None, source=paths.SAMPLE_DATA_ROOT, config=NO_NOISE
+    )
+    return formatted_1040
 
 
-@pytest.fixture(scope="module")
-def sample_data_social_security():
-    return _load_sample_data("social_security")
-
-
-@pytest.fixture(scope="module")
-def sample_data_taxes_w2_and_1099():
-    return _load_sample_data("taxes_w2_and_1099")
-
-
-@pytest.fixture(scope="module")
-def sample_data_taxes_1040():
-    return _load_sample_data("taxes_1040")
+@pytest.fixture(scope="session")
+def formatted_1040_sample_data_state_edit(request):
```

Comment:
Why do you need to use `request`, rather than being able just to pass the fixture `split_sample_data_dir_state_edit` directly?

Reply:
I'm getting the file path to the tmpdir factor for the split state data.